content/sql/concepts/commands/terms/contains/contains.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'CONTAINS'
+Description: 'Searches for matches to words or phrases.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'PostgreSQL'
+  - 'MySQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+CONTAINS() is a predicate used to search for matches to words or phrases in a column.
```

Comment:
```suggestion
In SQL, CONTAINS is a predicate used to search for matches to words or phrases in a column.
```

Reply:
Can we also mention something about the difference between this full-text search and the [LIKE](https://www.codecademy.com/resources/docs/sql/operators/like) operator?